### DISCUSSION TITLE
Current Chicago traffic data set does not include race, but I have a FOIA file includes race

### DISCUSSION THREAD

**Discussion by taitcha** [openpolicedata/openpolicedata#245] [General]
Hi, while looking at the data used by the Stanford Open Policing Project, the City of Chicago for traffic citations is listed, but race was not included. In 2018 (fulfilled in 2021) I requested a simliar data set on Chicago Police traffic citations that did include race, and it would be wonderful if this were added to the open police data set. You can see the file on Muckrock: https://www.muckrock.com/foi/chicago-169/traffic-citation-data-request-63311/?

**Comment by sowdm** [comment]
Hi @taitcha, thanks for bringing this to our attention. We are actively trying to make this type of FOIA data more easily accessible so this is really helpful. I took a quick look at the Chicago data from Stanford with our tool. It looks like there is a race column, but there are a lot of null values in it. I don't have time to look at it more deeply now, but I'm guessing this indicates that the original (and older) data acquired by Stanford did not have a race column (as reflected on Stanford's website) and that a race column was included for newer data in a later round of Stanford's FOIA requests. Stanford probably did not update their table after the data update. I can ping them about that.

I imagine that your dataset will fill in the gaps for the years that Chicago did not provide race data to Stanford. I'll have a deeper look at this tomorrow.

Are there any other datasets that you know of that would be useful to include in OPD?

**Comment by taitcha** [comment]
Great to hear! I don't have additional data sets to suggest unfortunately,
but hopefully this can be a quick addition to your database. Appreciate it!

Best,
Tait

**Comment by sowdm** [comment]
@taitcha Just added the data. It's not easy finding these datasets so really appreciate you pointing this one out! Thanks!

Data can now be accessed with OPD by the following:
```
import openpolicedata as opd
src = opd.Source("Chicago")
t = src.load_from_url(year="MULTIPLE",table_type='TRAFFIC CITATIONS')
t.table.head()
```

**Comment by taitcha** [comment]
Wonderful, thank you!

Best,
Tait
